templates/terraform/examples/ml_job_training.tf.erb

```diff
@@ -0,0 +1,12 @@
+resource "google_ml_engine_job" "test_job" {
```

Comment:
I'd recommend having two tests- a "basic" test where only required fields are set, and a "full" test that attempts to use as much of the resource as possible. Having a test that leaves most optional fields empty allows us to check  for optional fields whether we need to add a default value into api.yaml, mark it as default_from_api in terraform.yaml, or whether neither are necessary (I usually try to do neither until I see a test fail because we got a value back from the server that we didn't send).

Reply:
I will be coming with multiple test case

Reply:
test is added